langchain/llms/anthropic.py

```diff
@@ -16,7 +16,7 @@
 class _AnthropicCommon(BaseModel):
     client: Any = None  #: :meta private:
     async_client: Any = None  #: :meta private:
-    model: str = "claude-v1"
+    model: str = "claude-v2"
```

Comment:
I think this needs to be `claude-2`, not v2?

Reply:
Oh doh yes, thank you!